packages/expo-location/src/Location.ts

```diff
@@ -159,8 +171,39 @@ export async function enableNetworkProviderAsync(): Promise<void> {
 }
 
 export async function getCurrentPositionAsync(
-  options: LocationOptions = {}
+  options: LocationOptions = {},
+  methodOptions: LocationMethod = { type: LocationMethodType.Fast }
 ): Promise<LocationData> {
+  console.log(methodOptions);
+  if (methodOptions.type === LocationMethodType.Fast) {
+    console.log("fast");
+    return new Promise<LocationData>(async (resolve, reject) => {
+      try {
+        let bestPosition: LocationData | null = null;
+        let current = 0;
+        let { tries, minAccuracyRadius } = {
+          tries: 1,
+          minAccuracyRadius: 20000,
+          ...methodOptions,
+        };
+        let { remove } = await watchPositionAsync(options, position => {
+          current++;
+          if (bestPosition === null) {
```

Comment:
Here just check whether `bestPosition` is falsy
```suggestion
          if (bestPosition) {
```